src/binary-reader-ir.cc

```diff
@@ -1612,6 +1613,12 @@ Result BinaryReaderIR::ValidateDataSegmentSymbol(Index symbol_index) {
   return Result::Ok;
 }
 
+Result BinaryReaderIR::OnRelocCount(Index count, Index section_index) {
+  module_->elem_relocations_active_ =
+      section_index == static_cast<int>(BinarySection::Elem);
```

Comment:
The Elem section is the one that hold table elements (indirect function table).   Aren't you interested in the Data segement?

Reply:
I think you want to do something like in the objdump reader:

```
 reloc_section_ = section_types_[section_index]; 
```

Then elsewhere

```
if (reloc_section_ == BinarySection::Data)
```

Reply:
The type codes are completely standard.. but the index here is in section index with in the file.   So any file with just one section that section will have index of 0 ... regardless of its type.

Reply:
Yeah, I was wrong on both fronts.  It's Code and Data - and I need the objdump logic to make that happen.  (Because I wasn't doing it right I ended up with Elem and Start - which makes no sense.)

Fixed.